src/Views/DisplaysView.vala

```diff
@@ -0,0 +1,213 @@
+                    rotation_lock_grid.add (rotation_lock_label);
+                    rotation_lock_grid.add (rotation_lock_switch);
+                    
+                    action_bar.pack_start (rotation_lock_grid);
+
+                    touchscreen_settings.bind ("orientation-lock", rotation_lock_switch, "state", SettingsBindFlags.DEFAULT);
+                } else {
+                    info ("Schema \"org.gnome.settings-daemon.peripherals.touchscreen\" is not installed on your system.");
+                }
+            }
+
+            action_bar.pack_end (button_grid);
+
+            orientation = Gtk.Orientation.VERTICAL;
+            column_spacing = 6;
+            add (stack);
+            add (action_bar);
+            show_all ();
+
+            displays_overlay.configuration_changed.connect ((changed) => {
+                apply_button.sensitive = changed;
+            });
+
+            mirror_grid.sensitive = displays_overlay.active_displays > 1;
+            displays_overlay.notify["active-displays"].connect (() => {
+                mirror_grid.sensitive = displays_overlay.active_displays > 1;
+            });
+
+            mirror_display.configuration_changed.connect ((changed) => {
+                apply_button.sensitive = changed;
+            });
+
+            detect_button.clicked.connect (() => displays_overlay.rescan_displays ());
+            apply_button.clicked.connect (() => {
+                var rr_screen = new Gnome.RRScreen (Gdk.Screen.get_default ());
+                var rr_config = new Gnome.RRConfig.current (rr_screen);
+                if (rr_config.get_clone ()) {
+                    mirror_display.apply_configuration ();
+                } else {
+                    displays_overlay.apply_configuration ();
+                }
+
+                apply_button.sensitive = false;
+            });
+
+            var rr_screen = new Gnome.RRScreen (Gdk.Screen.get_default ());
+            var rr_config = new Gnome.RRConfig.current (rr_screen);
+            mirror_switch.active = rr_config.get_clone ();
+            if (rr_config.get_clone ()) {
+                stack.set_visible_child (mirror_display);
+            }
+
+            mirror_switch.notify["active"].connect (() => {
+                var rr_screen2 = new Gnome.RRScreen (Gdk.Screen.get_default ());
+                var rr_config2 = new Gnome.RRConfig.current (rr_screen2);
+                if (mirror_switch.active) {
+                    unowned Gnome.RRMode highest_mode = null;
+                    foreach (unowned Gnome.RRMode mode in rr_screen2.list_clone_modes ()) {
+                        if (highest_mode == null) {
+                            highest_mode = mode;
+                        } else if (mode.get_width () > highest_mode.get_width ()) {
+                            highest_mode = mode;
+                        }
+                    }
+
+                    if (highest_mode == null)
```

Comment:
According to our style guide braces are required even if there is only one line of code.